docs/supported-mutators.md

```diff
@@ -25,6 +25,7 @@ All Stryker versions support a variety of different mutators. We've aligned on a
 | [String Literal](#string-literal)                 |                     ✅                     |                        ✅                        |                      ✅                      |
 | [Unary Operator](#unary-operator)                 |                     ✅                     |                        ✅                        |                      ❌                      |
 | [Update Operator](#update-operator)               |                     ✅                     |                        ✅                        |                     n/a                      |
+| [Math Methods](#math-methods)                     |                     ❌                     |                        ✅                        |                      ❌                      |
```

Comment:
This shouldn't be a new mutator IMO. These are just `Method Expression` mutators, right?

Reply:
Well yes and no. We specifically check if the method is called on the Math class. If not we won't mutate. That's different from the other method mutators. We mutate those whatever they are called upon.

We could add these to the method expression mutators section. But then we should add `Math.` in front of all the mutations. And in Stryker.NET we already have them in a seperate section

Reply:
IMO this is something that should be able to turned off separately from the other method mutations.

Reply:
> IMO this is something that should be able to turned off separately from the other method mutations.

I can see that. That's why I think we should allow grouping outside of these groups and individually name each mutant operator as per #1458 (I know the PR isn't sexy, but please take a look and provide feedback. It becomes more and more critical by the day).

However, that shouldn't be a reason to separate them here. It's still a method expression being mutated.

> We specifically check if the method is called on the `Math` class.

That is smart, but a tiny detail shouldn't be steering the category naming scheme IMO.

In the end, I think:
1. Each mutant operator should have a unique mutation-testing-framework-agnostic name. For example: `AcosMethodCallNegation`
2. Each mutant operator fits into exactly one category. For example: `Method Expression`
3. Each mutant operator can fit into multiple groups. For example, based on level or class. For example `@level1` or `@mathMethods`

Reply:
For now I'd rather keep it like this. Since it has been merged already in Stryker.NET and would require a breaking change to revert. If in the future other strykers get math mutations as well we can realign.